src/wolfio.c

```diff
@@ -910,16 +916,28 @@ int wolfIO_TcpConnect(SOCKET_T* sockfd, const char* ip, word16 port, int to_sec)
             return -1;
         }
     }
+#else
+#if (__GLIBC__ >= 2) && defined(__USE_MISC) && !defined(SINGLE_THREADED)
+    ghbn_r_buf = (char *)XMALLOC(2048, NULL, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
Consider using an enum/define for 2048 here or finding one provided by existing headers. At a minimum please document how you came up with this value.